src/routers/alpha-router/alpha-router.ts

```diff
@@ -1957,16 +1917,28 @@ export class AlphaRouter
 
       quotePromises.push(
         Promise.all([v3CandidatePoolsPromise, v2CandidatePoolsPromise]).then(
-          ([v3CandidatePools, v2CandidatePools]) =>
-            this.mixedQuoter
+          async ([v3CandidatePools, v2CandidatePools]) => {
+            const crossLiquidityPools = await getMixedCrossLiquidityCandidatePools({
+              tokenIn,
+              tokenOut,
+              routingConfig,
+              v2PoolProvider: this.v2PoolProvider,
+              v3PoolProvider: this.v3PoolProvider,
+              v2SubgraphProvider: this.v2SubgraphProvider,
+              v3SubgraphProvider: this.v3SubgraphProvider,
+              v2Candidates: v2CandidatePools,
+              v3Candidates: v3CandidatePools,
+            });
+
```

Comment:
This is where we obtain the "missing pools" before calculating routes and quotes for mixed routes